openapi/v2/support-scheduler.yaml

```diff
@@ -128,6 +128,36 @@ components:
         start:
           description: "Start time in ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = \"yyyymmdd'T'HHmmss\")"
           type: string
+      required:
+        - name
+        - frequency
+    UpdateInterval:
+      description: "Defines the frequency at which some action should occur."
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        id:
+          description: "Uniquely identifies the interval"
+          type: string
+          format: uuid
+        name:
+          description: "Non-database identifier for an interval (*must be unique)"
+          type: string
+        start:
+          description: "Start time in ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = \"yyyymmdd'T'HHmmss\")"
+          type: string
+        end:
+          description: "Start time in ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = \"yyyymmdd'T'HHmmss\")"
+          type: string
+        frequency:
+          description: "The frequency of the event according ISO 8601"
+          type: string
+        runOnce:
+          description: "Indicates that this interval runs one time - at the time indicated by the start"
+          type: boolean
+      required:
+        - name
```

Comment:
Is name really required? I thought either name or Id is required.

Reply:
Modified.
You're right. The name or id is required for PATCH API.

Reply:
Added description.